api/v1beta1/vaultdynamicsecret_types.go

```diff
@@ -11,6 +11,8 @@ import (
 // VaultDynamicSecretSpec defines the desired state of VaultDynamicSecret
 type VaultDynamicSecretSpec struct {
 	// VaultAuthRef to the VaultAuth resource
+	// The VaultAuthRef can be prefixed with a namespace, eg: `namespaceA/vaultRefB`.
+	// If no namespace is specified the Operator will default to namespace of the VaultAuth CR.
```

Comment:
nit: this is a bit confusing since it's not possible to explicitly set the vaultAuth's namespace, wonder if we could say something like if the `namespace is included...` or if the `value is absolute`, not sure.

Reply:
Updated the wording for this in a52da2d